tests/console/consoletest_setup.pm

```diff
@@ -16,14 +16,16 @@
 use base "consoletest";
 use testapi;
 use utils;
+use ipmi_backend_utils 'use_ssh_serial_console';
 use strict;
 
 sub run {
     my $self = shift;
     # let's see how it looks at the beginning
     save_screenshot;
-
-    select_console 'root-console';
+    # use root-ssh for impi
+    check_var("BACKEND", "ipmi") ? use_ssh_serial_console : select_console 'root-console';
+    # select_console 'root-console';
 
```

Comment:
please rebase to upstream master first - the change in consoletest_setup is already merged